src/summary/level-two-routes.component.tsx

```diff
@@ -41,6 +42,11 @@ export const levelTwoRoutes: PatientChartRoute[] = [
     component: MedicationsDetailedSummary,
     name: "Medications"
   },
+  {
+    url: "/patient/:patientUuid/chart/medicationsleveltwo",
+    component: MedicationLevelTwo,
+    name: "Medication Details"
```

Comment:
```suggestion
    name: "Medications"
```

Reply:
We have an already existing name of medication

Reply:
The component currently there is a placeholder. You should be replacing this component with the one you're building.